chemicaltoolbox/autodock_vina/prepare_box/prepare_box.xml

```diff
@@ -1,12 +1,13 @@
-<tool id="prepare_box" name="Calculate the box parameters for an AutoDock Vina" version="0.1.0">
-    <description>job from an input mol file (confounding box)</description>
+<tool id="prepare_box" name="Calculate the box parameters for an AutoDock Vina" version="0.1.1">
+    <description>job from a ligand input file (confounding box)</description>
     <requirements>
-        <requirement type="package" version="2019.03.1.0">rdkit</requirement>
+        <requirement type="package" version="2019.03.1">rdkit</requirement>
```

Comment:
2019.03.4 is already out ... but we do not need to update now. Your call.

Reply:
I made a start but it seems this could be a rabbit hole... I will make a PR at some point with all tools and all packages updated (e.g. OpenBabel 3.0 is also out).